src/schedule-and-details/schedule-sidebar/__snapshots__/ScheduleSidebar.test.jsx.snap

```diff
@@ -39,7 +39,7 @@ exports[`<ScheduleSidebar /> should match the snapshot 1`] = `
         >
           <a
             className="pgn__hyperlink default-link standalone-link"
-            href="/course/course123/settings/details"
+            href="http://localhost:2001/course/course123/settings/details"
```

Comment:
Debating question. Do we need to set `http://localhost:2001` as a global constant for such a base URL?

Reply:
we already have `BASE_URL`